roles/eda/templates/eda.configmap.yaml.j2

```diff
@@ -26,7 +26,6 @@ data:
   # EDA Server
   EDA_ALLOWED_HOSTS: "['*']"
   EDA_CSRF_TRUSTED_ORIGINS: "http://{{ api_server_name }}:{{ api_nginx_port }}"
-  EDA_MQ_HOST: "{{ ansible_operator_meta.name }}-redis-svc"
```

Comment:
Why this line is removed? I think it is a mistake,

Reply:
No mistake!  It is now being set in the Redis configuration secret.  I should have removed it in https://github.com/ansible/eda-server-operator/pull/241 but I missed it.